src/tox_lsr/test_scripts/runcontainer.sh

```
@@ -269,66 +272,80 @@ yaml.safe_dump(val, open(sys.argv[2], "w"))
+        info committing container to create new image
         if ! podman container commit "$container_id" "$CONTAINER_IMAGE"; then
             podman rm -f "$container_id"
             return 1
         fi
+        info removing container
         podman rm -f "$container_id"
         image_name="$CONTAINER_IMAGE"
     fi
-    container_id=$(podman run -d "${CONTAINER_OPTS[@]}" ${LSR_CONTAINER_OPTS:-} \
+    info starting container
+    container_id=$(podman run -d "${initflags[@]}" "${CONTAINER_OPTS[@]}" ${LSR_CONTAINER_OPTS:-} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)